apps/user/src/components/fair/FairList/FairItem/FairStatus/FairStatus.tsx

```diff
@@ -0,0 +1,35 @@
+import { getFairStatus } from '@/utils';
+import { font } from '@maru/theme';
+import { flex } from '@maru/utils';
+import { CSSProperties } from 'react';
+import styled from 'styled-components';
+
+interface Props {
+    status: string;
+}
+
+const FairStatus = ({ status }: Props) => {
+    const { color, backgroundColor, fairStatus } = getFairStatus(status);
+
+    return (
+        <StyledFairStatus color={color} backgroundColor={backgroundColor}>
+            {fairStatus}
+        </StyledFairStatus>
+    );
+};
+
+export default FairStatus;
+
+const StyledFairStatus = styled.div<{
+    color: CSSProperties['color'];
+    backgroundColor: CSSProperties['backgroundColor'];
+}>`
+    ${font.context}
+    ${flex({ justifyContent: 'center', alignItems: 'center' })}
+    background-color: ${(props) => props.backgroundColor};
+    border: 1px solid ${(props) => props.color};
+    color: ${(props) => props.color};
+    border-radius: 100px;
+    height: 32px;
+    padding: 0px 10px;
```

Comment:
```suggestion
    padding: 0px 10px;
    white-status: nowrap;
```
처럼 `white-status: nowrap;` 속성을 추가해주세요.

<img width="1033" alt="스크린샷 2023-09-09 오후 11 30 55" src="https://github.com/Bamdoliro/marururu/assets/102217839/7838d12c-1c59-42b3-a9c6-30a52f4f56de">

최소 너비에서 글자가 줄바꿈됩니다.